clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -35,28 +35,29 @@
 
 import tachyon.Constants;
 import tachyon.TachyonURI;
+import tachyon.client.block.BSContext;
+import tachyon.client.file.FSContext;
 import tachyon.client.table.RawTable;
 import tachyon.conf.TachyonConf;
-import tachyon.master.MasterClient;
-import tachyon.thrift.ClientBlockInfo;
-import tachyon.thrift.ClientDependencyInfo;
-import tachyon.thrift.ClientFileInfo;
-import tachyon.thrift.ClientRawTableInfo;
-import tachyon.thrift.ClientWorkerInfo;
+import tachyon.thrift.DependencyInfo;
+import tachyon.thrift.FileBlockInfo;
+import tachyon.thrift.FileInfo;
+import tachyon.thrift.WorkerInfo;
 import tachyon.underfs.UnderFileSystem;
+import tachyon.util.ThreadFactoryUtils;
 import tachyon.util.io.FileUtils;
 import tachyon.util.network.NetworkAddressUtils;
 import tachyon.util.network.NetworkAddressUtils.ServiceType;
-import tachyon.util.ThreadFactoryUtils;
 import tachyon.worker.ClientMetrics;
 import tachyon.worker.WorkerClient;
 
 /**
  * Client API to use Tachyon as a file system. This API is not compatible with HDFS file system API;
  * while tachyon.hadoop.AbstractTFS provides another API that exposes Tachyon as HDFS file system.
- * Under the hood, this class maintains a MasterClient to talk to the master server and
+ * Under the hood, this class maintains a MasterClientBase to talk to the master server and
```

Comment:
We'd use use links ({@link ClassName}) in JavaDocs, to make it robust against refactoring, i.e. IDE can automatically update it.